application.md

```diff
@@ -1,153 +1,178 @@
 # Questionnaire
 
-## If you did not choose "veteran" in the checkbox, have you applied in the past? If so, for what year(s)?
+## Why does your org want to participate in Google Summer of Code?
+
+OpenAstronomy would like to be a part of Google Summer of Code to reach a new
+pool of contributors, develop our communities, and help the member projects
+collaborate better.
+Some of our member projects' past experience with GSoC shows that it provides a
+boost to areas which are in need of intensive attention, as well as adding
+members to the community who can turn out to be valuable long-lasting
+contributors.
+We would like to help bring these experience to a wider variety of projects, all
+united by the goal of fostering open source development of astronomy software.
 
-This organisation is new and this is our first year applying as such, however,
-two of our sub-organisations (Astropy and SunPy) have participated in GSoC under
-the Python Software Foundation umbrella, both in 2013 and 2014.
-
-## Why is your organization applying to participate in Google Summer of Code 2015? What do you hope to gain by participating?
-
-OpenAstronomy is a collection of projects united by a common goal to foster open
-source development of astronomy software. 
 We are submitting this application together in order to strengthen collaboration
 between the different projects, for example by offering project ideas that
 extend across multiple packages, in order to reduce duplication of efforts and
-strengthen interoperability. 
-As an umbrella organization separate from the Python Software Foundation, we
-will be able to include other projects that are not exclusively Python-based and
-accept students with experience with different languages, while uniting projects
-with the common goal of providing tools for the scientific exploration of the
-Universe.
-
-By participating in GSoC as a group, OpenAstronomy aims to improve the
+strengthen interoperability.
```

Comment:
The sentences below are stronger and communicate similar information, I would delete this sentence in the interest of space.